Source/WebCore/html/canvas/WebGLExtension.h

```diff
@@ -96,6 +96,8 @@ class WebGLExtensionBase : public RefCounted<WebGLExtensionBase> {
 public:
     WebGLExtensionName name() const { return m_name; }
 
+    virtual WebGLExtensionBase() = default;
```

Comment:
Shouldn't this be `~ WebGLExtensionBase` instead of `WebGLExtensionBase`?

Reply:
Ah, this patch is uploaded from different machine, so typo, fixed.